grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -428,6 +428,21 @@ class ProductAvailabilityService {
         return availableItems
     }
 
+    // Include also bin locations with negative qty (needed for edit page items)
+    List<AvailableItem> getAllAvailableBinLocations(Location location, List products) {
```

Comment:
Either add the same for single product (and call it inside this on) or make it for a single product instead of list.

Reply:
@awalkowiak Done.